### DISCUSSION TITLE
Unable to parse translation - Invalid plural category while building

### DISCUSSION THREAD

**Discussion by gabord** [CaiJimmy/hugo-theme-stack#413] [Q&A] [answered]
Hi,
I'm running into a problem while building with this theme and I can't seem to figure out what I'm missing here.
When building (`hugo serve` or just `hugo`), it randomly throws these errors, like it cannot parse nested yml properties:

`Error: add site dependencies: load resources: loading translations: "C:\dev\blog\blog\themes\hugo-theme-stack\i18n\en.toml:1:1": failed to load translations: unable to parse translation #1 because invalid plural category page
map[id:list translation:map[page:map[one:{{ .Count }} page other:{{ .Count }} pages] section:map[other:Section] subsection:map[one:Subsection other:Subsections]]]`

`Error: add site dependencies: load resources: loading translations: "C:\dev\blog\blog\themes\hugo-theme-stack\i18n\en.toml:1:1": failed to load translations: unable to parse translation #0 because invalid plural category archives
map[id:widget translation:map[archives:map[more:map[other:More] title:map[other:Archives]] tagCloud:map[title:map[other:Tags]]]]`

Tried to convert it to toml as well, but it results the same, as you can see.
This is the 'en' translation, but it actually started from the first one that was in the directory ('de'), I deleted all translations except English. 

On Windows 11 Pro, tried with `0.89.1`, `0.89.1-extended`, `0.87.0` and `0.87.0-extended`. 

Thank your for your ideas!

**Comment by gabord** [comment, marked as answer]
Oops, turned out, there was already a Chocolately-installed Hugo from earlier on the path :D

**Comment by radityaarhya** [comment]
having a same problem when running on Cloudflare Pages, but normal on Local. Any suggestions?

**Reply by CaiJimmy** [reply]
Set Hugo version to the lastest using environment variables: https://developers.cloudflare.com/pages/framework-guides/deploy-a-hugo-site/#using-a-specific-hugo-version

**Reply by radityaarhya** [reply]
thanks, really helpful!
